server/api/student/student.model.js

```diff
@@ -43,6 +43,17 @@ const studentSchema = new Schema({
         type: Number,
         min: 0,
         max: 100
+    },
+    social: {
+        type: Number,
+        min: 1,
+        max: 4
+    },
+    prefer: {
+        type:{
```

Comment:
If the label was meant to represent hard coded 'first', 'second'...
I think it's better to actually create the prefer object like this:
```js
prefer: {
  first: {
    id: String
  },
 second: {
  },
 third: {
  }
}
```